src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -664,9 +665,22 @@ export default class AircraftCommander {
     runIls(aircraft, data) {
         const approachType = 'ils';
         const runwayName = data[1].toUpperCase();
-        const runway = AirportController.airport_get().getRunway(runwayName);
+        const runwayModel = AirportController.airport_get().getRunway(runwayName);
+        const glideslopeElevation = runwayModel.getGlideslopeAltitudeAtFinalApproachFix();
+        const minmiumGlideslopeInterceptAltitude = _ceil(glideslopeElevation, -2);
 
-        return aircraft.pilot.conductInstrumentApproach(approachType, runway);
+        if (aircraft.mcp.altitude < glideslopeElevation) {
+            const readback = {};
+
+            readback.log = `unable ILS ${runwayModel.name}, our assigned altitude is below the minimum ` +
```

Comment:
I feel like all of this should be part of `Pilot.conductInstrumentApproach`

Reply:
i know you're right. i guess i was avoiding it because we've been talking about getting rid of the pilot. Moved this to the pilot and updated the requisite tests.